### PR TITLE
Update dynamic.py

### PR DESCRIPTION
**Pull request by robbeofficial**
allow non-ascii chars in MSG files

### CODE REVIEW

File: src/genpy/dynamic.py [hunk 1]
```diff
@@ -165,7 +165,8 @@ def generate_dynamic(core_type, msg_cat):
     
     # write the entire text to a file and import it (it will get deleted when tmp_dir goes - above)
     tmp_file = tempfile.NamedTemporaryFile(suffix=".py",dir=tmp_dir,delete=False)
-    tmp_file.file.write(full_text.encode())
+    #tmp_file.file.write(full_text.encode())
```
Comment (dirk-thomas) [inline src/genpy/dynamic.py]: Please remove this line since the next line replaces it. The git history will be sufficient.

**Comment by dirk-thomas** [comment]
We are aiming for a new round of releases early next week (Mon or Tue). It would be great f this would be ready before in order to get released.

**Comment by moooeeeep** [comment]
You can add something like this to [test_genpy_dynamic.py](https://github.com/ros/genpy/blob/kinetic-devel/test/test_genpy_dynamic.py) to have a unit test for this use case:

    try:
        msgs = generate_dynamic("gd_msgs/MyAngle", "float32 angle # \xc2\b0 \n")
    except UnicodeDecodeError:
        assert False, "Can't handle UTF-8 in comments"

**Comment by dirk-thomas** [comment]
@moooeeeep Please provide a pull request for proposed code changes.

**Comment by peteflorence** [comment]
@dirk-thomas can I open a new PR and get this merged?
This just burned me so bad having a unicode character in a ROS msg comment :), see: https://github.com/nalt/wsg50-ros-pkg/issues/10

or @robbeofficial can you push this through?

**Comment by dirk-thomas** [comment]
> can I open a new PR

Sure.

> get this merged?

This will depend on another review but if it works as expected in Python 2 and 3 as well as doesn't break anything then it can be merged.

**Comment by dirk-thomas** [comment]
Closing due to no activity.

**Pull request closed without merging**